packages/shared/src/components/modals/NewSourceModal.tsx

```diff
@@ -184,8 +183,10 @@ export default function NewSourceModal(props: ModalProps): ReactElement {
           onSubmit={onScrapeSubmit}
           aria-busy={isScraping}
           data-testid={`login state: ${loginState?.trigger}`}
+          id="submit-source"
```

Comment:
What are we using the ID for?

Reply:
Here:
https://github.com/dailydotdev/apps/pull/1404/files#diff-2b63e881f1f68e2233cfd08a081ff1d03677b0b621b65ad9e8dc7ccc4777369eR264